extensions/dataseed/dataseed-aws/src/main/java/com/microsoft/dagx/dataseed/aws/S3BucketDataseedExtension.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ *  All rights reserved.
+ *
+ */
+
+package com.microsoft.dagx.dataseed.aws;
+
+import com.microsoft.dagx.spi.monitor.Monitor;
+import com.microsoft.dagx.spi.system.ServiceExtension;
+import com.microsoft.dagx.spi.system.ServiceExtensionContext;
+import com.microsoft.dagx.transfer.provision.aws.provider.ClientProvider;
+import software.amazon.awssdk.core.async.AsyncRequestBody;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.s3.S3AsyncClient;
+import software.amazon.awssdk.services.s3.model.CreateBucketRequest;
+import software.amazon.awssdk.services.s3.model.PutObjectRequest;
+import software.amazon.awssdk.services.s3.model.PutObjectResponse;
+
+import java.io.IOException;
+import java.util.Set;
+import java.util.concurrent.CompletableFuture;
+
+public class S3BucketDataseedExtension implements ServiceExtension {
+
+    private static final String REGION = Region.US_EAST_1.toString();
+    private static final String BUCKET_NAME = "dagx-src-bucket";
+    private static final String TEST_FILE_NAME = "donald.png";
+    private static final String TEST_FILE_NAME2 = "donald.png";
+    private Monitor monitor;
+    private S3AsyncClient s3Client;
+
+    @Override
+    public void initialize(ServiceExtensionContext context) {
+
+        var clientProvider = context.getService(ClientProvider.class);
+        monitor = context.getMonitor();
+        s3Client = clientProvider.clientFor(S3AsyncClient.class, REGION);
+
+
+        monitor.info("AWS DataSeed: uploading a few test files");
+        var is = getClass().getClassLoader().getResourceAsStream(TEST_FILE_NAME);
```

Comment:
Maybe extract a method which can load resource files and call it twice by resource name